benchmarks/discrete_trajectory.cpp

```diff
@@ -63,8 +63,8 @@ DiscreteTrajectory<World> MakeTrajectory(Timeline<World> const& timeline,
         double const split = *it_split;
         CHECK_LE(0, split);
         CHECK_LE(split, 1);
-        t_split = Barycentre<Instant, double>({t_min, t_max},
-                                              {1 - split, split});
```

Comment:
I'd rather not change the formatting (we don't systematically use clang-format).

Reply:
Whoops. Fixed.